.ci/docker/node-container/Dockerfile

```diff
@@ -0,0 +1,19 @@
+ARG NODE_VERSION=8
+FROM node:${NODE_VERSION}
+
+WORKDIR /npm
+RUN ( [ "${NODE_VERSION%%.*}" -le 8  ] \
+  && echo "Node.js ${NODE_VERSION} - Manual install npm" \
+  && mkdir -p /npm/node_modules \
+  && npm install npm \
+  && rm /usr/local/bin/npm \
+  && ln -s /npm/node_modules/npm/bin/npm-cli.js /usr/local/bin/npm ) || exit 0
+
+RUN ( [ "${NODE_VERSION%%.*}" -gt 8  ] \
+  && echo "Node.js ${NODE_VERSION} - upgrade npm" \
+  && npm install -g npm@latest) || exit 0
```

Comment:
Reminder we're planning on dropping pre-8.x versions soon and therefore should remember to clean this up right away, if we land this first.

Reply:
I think this was carried over from the agent as well. We don't need to upgrade npm in any other repo than on the agent repo.

Reply:
Ah, right. In that case, we should just remove the upgrade altogether, since that would more closely match the configuration of most users.

Reply:
Alright, I'll remove the `npm install`. In fact, this particular implementation was taken from the agent, as Thomas mentioned.